app/models/unit.rb

```diff
@@ -47,4 +52,8 @@ def self.associations_adding_requirements
   def self.associations_adding_services
     [:building]
   end
+
+  def self.target_population_select_array
```

Comment:
Generally we'll name these something like `available_target_populations`  That way when it becomes a hash at a later date we don't have to rename it.